tests/coprocessor/test_select.rs

```diff
@@ -1508,19 +1507,12 @@ fn handle_streaming_select<F>(
 where
     F: FnMut(&Response) + Send + 'static,
 {
-    let (ftx, frx) = futures_mpsc::channel(16);
-    let callback = box move |s: CopStream<Response>, executor: Option<CpuPool>| {
-        let f = s.forward(ftx);
-        if let Some(executor) = executor {
-            return executor.spawn(f).forget();
-        }
-        f.wait().unwrap();
-    };
-    let req = RequestTask::new(req, OnResponse::Streaming(callback), 100).unwrap();
+    let (stream_tx, stream_rx) = futures_mpsc::channel(10);
+    let req = RequestTask::new(req, OnResponse::Streaming(stream_tx), 100).unwrap();
     end_point.schedule(EndPointTask::Request(req)).unwrap();
 
     let (tx, rx) = mpsc::channel();
```

Comment:
Wd don't need the `channel` exactly. We can build the `Vec` in `for` loop directly.

Reply:
any update?

Reply:
@hicqu Updated